### DISCUSSION TITLE
Import History from google takeout problem usnig JSON and CSV file

### DISCUSSION THREAD

**Discussion by cwsooner** [FreeTubeApp/FreeTube#5554] [General]
I can't get history uploaded into FreeTube using the Json file. When I go to the takeout file and hit select to upload, it just sits there with the select button still selected.

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
Hi @cwsooner what OS are you using? The import is happening in the background and can take a while if you have a large history to import.

**Comment by cwsooner** [comment]
Linux Mint 22

Sent with [Proton Mail](https://proton.me/) secure email.

**Comment by cwsooner** [comment]
I just checked and the history still hasn't uploaded.

Sent with [Proton Mail](https://proton.me/) secure email.

**Reply by efb4f5ff-1298-471a-8973-3d47447115dc** [reply]
We are working on a fix for it!

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
https://github.com/FreeTubeApp/FreeTube/pull/5666 was merged. I hope this addressed the issue you're facing. This will be available in our next release.
